resources/META-INF/plugin_template.xml

```diff
@@ -237,6 +237,8 @@
     <iconProvider implementation="io.flutter.project.FlutterIconProvider" order="first"/>
 
     <library.type implementation="io.flutter.sdk.FlutterPluginLibraryType"/>
+
+    <projectStructureDetector implementation="io.flutter.project.FlutterProjectStructureDetector"/>
```

Comment:
Can you give this an id attr? If possible, it might make adding an Android-specific one easier. If a bunch of others are already defined and none of them have id's then it isn't worth the trouble.

Reply:
It doesn't look like this is done in general but I'm happy to add one if it ends up coming in handy.